acceptance/gem_versions/.kitchen.yml

```diff
@@ -0,0 +1,47 @@
+driver:
+  name: ec2
+  instance_type: m3.medium
+  aws_ssh_key_id: serdar
```

Comment:
Totally. As we get more progress on the Jenkins tasks we should totally have it.